docs: preview/experimental docs in README.md (#2179)
Kong/kubernetes-ingress-controller: README.md

If you've spotted a bug, please open an issue
on our [Github](https://github.com/kong/kubernetes-ingress-controller/issues).

### Preview and Experimental Features

At any time the KIC may include features or options that are considered

> experimental and wont be enabled by default, nor available in the [Kong

Reviewer: Original was missing apostrophe, may as well also agree in tense with the preceding clause.
```suggestion
experimental and are not enabled by default, nor available in the [Kong
```

Documentation Site][kongdocs].

If you're a developer/enthusiast who's interested in testing or helping to

> develop new features you can get started by seeing the currently available

Reviewer 2: Don't add such an if condition here - unnecessary friction - click-through rate will drop as users could interpret this as "do not use warning".
Rather how about:
```
To try out new features that are behind feature gates, please see this
```

Author: ```suggestion
To try out new features that are behind feature gates, please see
```
